cli/jobs/single-step/r/iris/src/aml_utils.R

```diff
@@ -0,0 +1,92 @@
+# AML utility to extend support for MLflow in Azure Machine Learning (AML). This utility does the following:
```

Comment:
dangerous keeping 2 copies...

Reply:
@lostmygithubaccount any examples of how common utils can be used to avoid duplication across examples ?

Reply:
Would it be worth putting these functions into a small R package equivalent to the azureml-mlflow python package? I did that here: https://github.com/sdonohoo/azureml.mlflow because I didn't like having to repeatedly source the helper function files in multiple examples. Thinking you could execute clean_tracking_uri() onLoad in the package.

Reply:
Thanks @sdonohoo for the suggestion. Certainly a good option. Long term goal is to have utils code to move to Mlflow R OSS. With that in mind for now we agreed to keep it in examples and will finally transition it out to OSS sdk.